Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -1304,11 +1313,20 @@ final class StatusTableViewController: ChartsTableViewController {
                 UIApplication.shared.open(url)
             }
         }
-        
-        if let _ = sender.view as? CGMStatusContainerHUDView,
-            let cgmManager = deviceManager.cgmManager,
-            let cgmManagerUI = cgmManager as? CGMManagerUI
-        {
+    }
+    
+    @objc private func pumpStatusTapped( _ sender: UIGestureRecognizer) {
+        if let pumpManagerUI = deviceManager.pumpManager {
+            var completionNotifyingVC = pumpManagerUI.settingsViewController()
+            completionNotifyingVC.completionDelegate = self
+            self.present(completionNotifyingVC, animated: true, completion: nil)
+        } else {
+            performSegue(withIdentifier: SettingsTableViewController.className, sender: toolbarItems![8])
```

Comment:
Going to general settings after clicking a specific action "add pump" seems like it would cause user confusion. I think we should pop up the new device selection screen directly from this action, or directly to the first setup page of the pump manager if there is only one possible device to choose.

Reply:
Yeah, I agree. I asked this of the design team, but I haven't got an answer back. If it is alright with you, I'll leave it for their review and make sure that an answer is given.

Reply:
Design suggested starting the workflow directly, since we currently only support 1 cgm/pump. When we support more, behaviour will allow the user to select a specific cgm/pump.